README.md

```diff
@@ -1,25 +1,25 @@
-# ssp-client
+# rsp-client
 [![Build Status](https://travis-ci.org/redhat-developer/rsp-client.svg?branch=master)](https://travis-ci.org/redhat-developer/rsp-client)
-A simple client for the Simple Server Protocol (SSP) written in typescript.
+A simple client for the Runtime Server Protocol (RSP) written in typescript.
 
 Implements the API described [here](https://github.com/robstryker/org.jboss.tools.ssp/blob/master/api/src/main/java/org/jboss/tools/ssp/api/SSPClient.java) and [here](https://github.com/robstryker/org.jboss.tools.ssp/blob/master/api/src/main/java/org/jboss/tools/ssp/api/SSPServer.java).
 
-See the [SSP repo](https://github.com/robstryker/org.jboss.tools.ssp) for further details about SSP.
+See the [RSP repo](https://github.com/robstryker/org.jboss.tools.ssp) for further details about RSP.
 
 ## Installing
 
 ```sh
-npm i ssp-client
+npm i rsp-client
 ```
 
 ## Usage
 Basic "short" usage example, for full list of available methods, see [usage](USAGE.md).
 
 ```typescript
-import { SSPClient, Protocol, ServerState } from "ssp-client";
+import { RSPClient, Protocol, ServerState } from "rsp-client";
```

Comment:
We can only merge these changes when the codebase also is incorporated with the name changes of SSP->RSP. Till that time, we need to hold off this PR. Once the codebase is reflected with all the changes, it makes sense to update the README accordingly.

Reply:
As the changes of https://github.com/redhat-developer/rsp-client/pull/26 are merged. We can merge this.